src/components/comment/article.tsx

```diff
@@ -54,6 +54,14 @@ const topBorder = css`
     `}
 `;
 
+const lineStyles = css`
+    margin-top: 5.2rem;
```

Comment:
What does this do? is it the same height as the cutout (image in front of the opinion lines)? Maybe they could share a variable?

Reply:
This is for the positioning of the Lines component in relation to top of document, in order to line up with bottom of image.
Apparently the Cutout component is constructed from an Img component, of which the parent is a <div>.
The style for it is:
`position: absolute;
    top: -48px;`
It is moving 48px above the parent. Since my Lines component is in a separate <div> tag below I dont think i can share any parameters in the style variables. Unless perhaps I place the Lines component inside and part of the Cutout component? This may make the article/comment more difficult to deconstruct in future? Placing in a span didn't seem to work with any styles, Thinking out loud.

Reply:
- Look into supporting dark mode for Lines component with @SiAdcock